### DISCUSSION TITLE
Multiple ColumnLayers with Mapbox

### DISCUSSION THREAD

**Discussion by Manoj1229** [visgl/deck.gl#5767] [Ideas]
Hi, 

Can anyone explain to/suggest to me that how to make multiple layers with Mapbox? 

How to add two ColumnLayers on the same map with Python?

Thanks

**Comment by ajduberstein** [comment]
See the example [here](https://pydeck.gl/gallery/column_layer.html). You can specify a list of layers as the first argument of the `Deck` object, e.g.:

```
r = pdk.Deck(
    [first_column_layer, second_column_layer],
    initial_view_state=view,
    tooltip=tooltip,
    map_provider="mapbox",
    map_style=pdk.map_styles.SATELLITE,
)
```

**Reply by Manoj1229** [reply]
Thanks, is there any possible way to add legends to it.
